### ISSUE TITLE
Windows context menu key doesn't work for disks

### ISSUE THREAD

**Issue by v-xuanzh** [microsoft/AzureStorageExplorer#1822]
**Storage Explorer Version:** 1.10.0
**Build:** [20190906.4](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3023890)
**Branch:** master
**Platform/OS:** Windows 10
**Architecture:** ia32/x64
**Regression From:** Not a regression

**Steps to reproduce:**
1. Expand 'Disks' node -> Open one resource group.
2. Create a new disk from a local VHD file.
3. Select the created disk then press menu key.
4. Check whether the context menu shows or not.

**Expect Experience:**

The context menu shows well.

**Actual Experience:**

The menu key doesn't work.

**Comment by JasonYeMSFT** [comment]
In addition to this bug, we have noticed that under some circumstances invoking the context menu through keyboard is broken in no matter which data explorer. Due to limited time, we are not going to fix all of them for this release. Instead, I have made a partial fix for the managed disks as well as a work around. Given the work, we will have one work around for keyboard context menu access, which is to invoke the context menu through ctrl-5 (instead of shift-F10 or Menu key).

I have logged a separate bug which will be addressed in 1.11. Could you please temporarily ignore the broken context menu access and verify that the ctrl-5 work around works? Thanks.

**Comment by v-xuanzh** [comment]
Hi @JasonYeMSFT ,Thanks for your fixes for the managed disks. 

And we have noticed the new separate bug [1825](https://github.com/microsoft/AzureStorageExplorer/issues/1825). We will verify the ctrl-5 works fine once fixed it. :)
